config/test.yml.erb

```diff
@@ -33,6 +33,7 @@ netsim_enable_metrics: true
 stub_school_data: true
 daemon: true
 use_pusher: <%=!ci_test%>
+build_i18n: true
```

Comment:
please add a comment explaining that we want to install i18n dependencies in test so that we can run some i18n-specific tests

Reply:
done